docs/package.json

```diff
@@ -5,7 +5,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "clean": "rm -rf .next"
+    "clean": "rm -rf .next",
+    "format": "prettier --write \"**/*.{js,ts,tsx,md}\""
```

Comment:
Format:check script could be added for linting

Reply:
done